presto-main/src/main/java/com/facebook/presto/operator/aggregation/ApproximateAverageAggregations.java

```diff
@@ -79,15 +80,15 @@ public static void combine(ApproximateAverageState state, ApproximateAverageStat
         }
     }
 
-    @OutputFunction(StandardTypes.VARCHAR)
+    @OutputFunction(VARCHAR_MAX_LENGTH)
```

Comment:
I don't think max_length is appropriate here. This outputs a varchar resulting from the `formatApproximateAverage` function, so you should be able to determine the max length of that